djangogirls/urls.py

```diff
@@ -11,6 +11,7 @@
 
     # Regular links:
     url(r'^admin/', include(admin.site.urls)),
+    url(r'^jobs', include('jobs.urls', namespace='jobs')),
```

Comment:
we need the slash after jobs 
url(r'^jobs/'...
Then if the url entered is /jobs/, it will match.  If it's /jobs, it won't match any pattern and django will then try appending the slash and will match /jobs/  (without the slash, it will only match /jobs)